newIDE/app/package.json

```diff
@@ -100,6 +101,8 @@
     "extract-all-translations": "node scripts/extract-all-translations.js",
     "compile-translations": "node scripts/compile-translations.js",
     "reload-extensions": "node scripts/import-GDJS-Runtime.js",
+    "build-theme-resources": "node scripts/build-theme-styles.js",
```

Comment:
Let's really use the exact same name: `build-theme-resources` for both?
("resources" is a bit more generic than styles, but it's better because there is both a css file and a json file generated?)

Reply:
done ✅